sci-physics/fastjet/fastjet-3.4.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{8..10} )
+DOCS_BUILDER="doxygen"
+# we depend on these similar to putting doxygen[dot] in BDEPEND
+DOCS_DEPEND="media-gfx/graphviz media-libs/freetype"
+
+inherit autotools flag-o-matic fortran-2 python-r1 docs
+
+DESCRIPTION="A software package for jet finding in pp and e+e- collisions"
+HOMEPAGE="https://fastjet.fr/"
+SRC_URI="https://fastjet.fr/repo/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="cgal doc examples python +plugins +contrib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# For now, we depend on cgal[gmp(-)] and append -lgmp to libs, despite possible incompatibility with cgal-5.5
+DEPEND="
+	contrib? ( sci-physics/fastjet-contrib )
```

Comment:
The `contrib` use flag doesn't really do anything apart from pulling in fastjet-contrib. We usually avoid introducing use flags for these kind of situations because toggling this flag will then result in a useless recompile that doesn't changed the installed files. Additionally the flag causes a circular dependency so I dropped it.